lib/components/leftNavigation/leftNavigation.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Menu as AntDMenu } from 'antd';
+import styles from './leftNavigation.css';
+
+const LeftNavigation = (props) => {
+  const { children } = props;
+  const [collapsed, setCollapsed] = useState(true);
+
+  return (
+    <div className={styles.menuWrapper}>
+      <AntDMenu
+        className={styles.leftMenu}
+        mode="inline"
+        inlineCollapsed={collapsed}
+        onMouseEnter={() => setCollapsed(false)}
+        onMouseLeave={() => setCollapsed(true)}
+        onFocus={() => setCollapsed(false)}
+        onBlur={() => setCollapsed(true)}
```

Comment:
Thinking about this a little more, this logic might actually need to get a little messy.

What if we `mouseEnter`, then tab through the document, eventually `blur`ing this menu. Will that collapse the menu while hovered? It may be an accessibility issue.